manipulador:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  
  
  # Position Controllers ---------------------------------------
  first_controller:
    type: effort_controllers/JointPositionController
    joint: first
    pid: {p: 1000.00, i: 200.00, d: 200.00}

  second_controller:
    type: effort_controllers/JointPositionController
    joint: second
    pid: {p: 1000.00, i: 200.00, d: 200.00}

  third_controller:
    type: effort_controllers/JointPositionController
    joint: third
    pid: {p: 1000.00, i: 200.00, d: 200.00}

  fourth_controller:
    type: effort_controllers/JointPositionController
    joint: fourth
    pid: {p: 1000.00, i: 200.00, d: 200.00}

  fifth_controller:
    type: effort_controllers/JointPositionController
    joint: fifth
    pid: {p: 1000.00, i: 0.100, d: 0.00}
